hass-addon-sunsynk-multi/run.py

```diff
@@ -126,6 +133,7 @@ def setup_driver() -> None:
             "server_id": opt.modbus_id,
             "timeout": OPT.timeout,
             "read_sensors_batch_size": OPT.read_sensors_batch_size,
+            "dongle_serial_number": opt.dongle_serial_number,
```

Comment:
Should you not add this only if its the solarman driver?

or create a driver_specific dict (say on line 105) that you merge here? Then you can add solarmansunsynk speciifc options where you assign the factory var on line 121

```suggestion
            **driver_specific_options
```

Reply:
I've made this change